From 8c53fe98548f7185488ceb1b8c2514add663b02b Mon Sep 17 00:00:00 2001
From: Yuantian Tang <andy.tang@nxp.com>
Date: Thu, 29 Aug 2019 15:16:26 +0800
Subject: [PATCH 063/245] arm64: dts: ls1028a: fix a compatible issue

commit a008bb9d6b1852ad06bc552bb70e1d21709142d2 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

The I2C multiplexer used on ls1028aqds is PCA9547, not PCA9847.
So correct it.

Signed-off-by: Yuantian Tang <andy.tang@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
index f4f95336b485..1931c059ccc5 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
@@ -115,7 +115,7 @@
 	status = "okay";
 
 	i2c-mux@77 {
-		compatible = "nxp,pca9847";
+		compatible = "nxp,pca9547";
 		reg = <0x77>;
 		#address-cells = <1>;
 		#size-cells = <0>;
-- 
2.17.1

